FoxHound.Client/src/Comments/Comment.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {CommentModel} from "./Comments";
+import {makeStyles} from "@material-ui/core/styles";
+import {Paper, Container} from "@material-ui/core/";
+
+interface IProps {
+    comment : CommentModel;
+}
+
+const Comment : React.FC < IProps > = (props) => {
+    const useStyles = makeStyles((theme) => ({
+        root: {
+            padding: theme.spacing(2, 3),
+            marginBottom: '10px'
+        }
+    }));
```

Comment:
We've been using a mix of the the useStyle with a root class and using the "<Box"> component. We should standardize on how we want to do this at some point.